Integrations/integration-FreshDesk.yml

```diff
@@ -0,0 +1,2574 @@
+        filter = args.get('filter')
+        if filter:
+            if filter == 'deleted':
+                additional_fields.append('deleted')
+            elif filter == 'spam':
+                additional_fields.append('spam')
+        requester = args.get('requester')
+        if requester:
+            if '@' in requester:
+                additional_fields.append('email')
+        company_id = args.get('company_id')
+        if company_id:
+            additional_fields.append('company_id')
+        if args.get('include_description') and args.get('include_description').lower() == 'yes':
+            additional_fields.extend(['description', 'description_text'])
+        return additional_fields
+
+
+    def entries_to_files(entry_ids):
+        """
+        Format file details (retrieved using the files' entry IDs) to API expectations to include files in API call.
+        """
+        attachments = []
+        for entry_id in entry_ids:
+            execute_results = demisto.getFilePath(entry_id)
+            file_path = execute_results["path"]
+            file_name = execute_results["name"]
+            attachments.append(('attachments[]', (file_name, open(file_path, 'rb'))))
+
+        return attachments
+
+
+    def handle_array_input(args):
+        """
+        Format any command argument that is supposed to be an array from a string to a list.
+        """
+        array_inputs = [
+            'tags', 'attachments', 'cc_emails', 'bcc_emails', 'to_emails',
+            'update_fields', 'update_values', 'notify_emails'
+        ]
+        attchs_present = args.get('attachments')
+        if attchs_present:
+            for arr_input in array_inputs:
+                if arr_input in args.keys():
+                    if arr_input != 'attachments':
+                        args[arr_input + '[]'] = argToList(args.get(arr_input))
+                        del args[arr_input]
+                    else:
+                        args[arr_input] = argToList(args.get(arr_input))
+        else:
+            for arr_input in array_inputs:
+                if arr_input in args.keys():
+                    args[arr_input] = argToList(args.get(arr_input))
+        return args
+
+
+    def handle_bool_input(args):
+        """
+        Format entered values for command arguments 'private' and 'incoming' to API expectations.
+        """
+        bool_inputs = ['private', 'incoming']
+        acceptable_vals = ['true', 'false']
+
+        try:
+            for bool_input in bool_inputs:
+                if bool_input in args.keys():
+                    args[bool_input] = args.get(bool_input).lower()
+                    if args[bool_input] not in acceptable_vals:
+                        raise ValueError('field: "{}"\n'.format(bool_input))
+        except ValueError as e:
+            return_error(e.message + 'The only valid inputs for this field are either "True" or "False"')
```

Comment:
don't think we need this validation.
if we give the user a dropdown with the possible values, it's enough